synapse/storage/databases/main/search.py

```diff
@@ -33,7 +33,7 @@
 
 
 class SearchWorkerStore(SQLBaseStore):
-    def store_search_entries_txn(self, txn, entries):
+    def store_search_entries_txn(self, txn, entries: Iterable[SearchEntry]):
```

Comment:
```suggestion
    def store_search_entries_txn(self, txn, entries: Iterable[SearchEntry]) -> None:
```

And please remove the type for `entries` in the docstring (and probably move up the description to a single line).

Reply:
I think you want to apply ^

Reply:
also annotate `txn: LoggingTransaction` whilst we're here, if you don't mind :)

Reply:
also remove the types from the docstring `txn (cursor):` → `txn:` etc — we tend to do that to make sure they don't fall out of sync with the signature that is checked by Mypy